### PR TITLE
Deploy API server via SSH commands

### PR DESCRIPTION
**Pull request by a-chabin**

### CODE REVIEW

File: .github/workflows/api-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      with:
+        host: ${{ secrets.API_SERVER_HOST }}
+        username: ${{ secrets.API_SERVER_USERNAME }}
+        password: ${{ secrets.API_SERVER_PASSWORD }}
+        script: |
+          # Fix NVM appleboy/ssh-action
+          export NVM_DIR=~/.nvm
+          source ~/.nvm/nvm.sh
+
+          export APP_PORT=${{ secrets.API_SERVER_PORT }}
```
Comment (hiba9201) [inline .github/workflows/api-deploy.yml]: I feel pain :)

File: .github/workflows/api-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          cd transport-api-test.na4u.ru/transport
+          git fetch
+          git checkout origin/test-deploy
```
Comment (hiba9201) [inline .github/workflows/api-deploy.yml]: we're gonna leave it like that?

**Review by hiba9201 — APPROVED** [review]

File: .github/workflows/api-deploy.yml (hunk 2 above)
Comment (hiba9201) [reply .github/workflows/api-deploy.yml]: forget this question)

File: .github/workflows/api-deploy.yml (hunk 1 above)
Comment (a-chabin) [reply .github/workflows/api-deploy.yml]: 🤧

**Pull request merged**